src/staging/src/pages/sitemap.js

```diff
@@ -4,21 +4,38 @@ import MegaMenu from "../components/megaMenu.js"
 import { Helmet } from "react-helmet"
 import Header from "../components/header"
 import Footer from "../components/footer"
+import { Link, Breadcrumbs } from "@material-ui/core"
```

Comment:
Does Breadcrumbs require Link as well? If not can we use the Gatsby Link component here instead?

As discussed fine to use the MUI breadcrumbs for now but let's add a TODO somewhere in this file

Reply:
done